docs/user-guide_installation.md

```diff
@@ -44,7 +46,7 @@ Select the following tabs for the installation description of your target.
 
     Linux users will need to download and install `p7zip` (the terminal version of `7zip`).
 
-    ??? info "How do I extract DietPi image on Linux"
+    ??? hint "How do I extract DietPi image on Linux"
```

Comment:
Great. I did not know the 'hint' keyword. :-)

Reply:
Glad you like it. I thought it fit better the context.